### PR TITLE
[14.0][MIG] web_group_expand

### PR DESCRIPTION
**Pull request by BT-aestebanez**
Buttons moved from pager to buttons
image

### CODE REVIEW

**Review by ghost — APPROVED** [review]
LGTM

**Review by BT-panken — APPROVED** [review]
LGTM

File: web_group_expand/i18n/ca.po [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* web_group_expand
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```
Comment (BT-cserra) [inline web_group_expand/i18n/ca.po]: `Odoo Server 14.0\n`? Same for the other PO files.

File: web_group_expand/i18n/web_group_expand.pot [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* web_group_expand
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 13.0\n"
```
Comment (BT-cserra) [inline web_group_expand/i18n/web_group_expand.pot]: `Odoo Server 14.0\n`?

File: web_group_expand/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+* Mantavya Gajjar <mga@openerp.com>
+* Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>
+* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+* Jay Vora (SerpentCS) for their alternative implementation
+* Jan Verbeek <jverbeek@therp.nl>
+* Manuel Calero <manuelcalerosolis@gmail.com>
+* Alvaro Estebanez (Brain-tec) <alvaro.estebanez@bt-group.com>
```
Comment (BT-cserra) [inline web_group_expand/readme/CONTRIBUTORS.rst]:
```suggestion
* Alvaro Estebanez (brain-tec AG) <alvaro.estebanez@bt-group.com>
```

**Review by BT-cserra — APPROVED** [review]
Minor thing regarding the version in the PO files. Pre-approved.

File: web_group_expand/i18n/ca.po (hunk 1 above)
Comment (BT-aestebanez) [reply web_group_expand/i18n/ca.po]: done, thanks

File: web_group_expand/i18n/web_group_expand.pot (hunk 2 above)
Comment (BT-aestebanez) [reply web_group_expand/i18n/web_group_expand.pot]: done, thanks

File: web_group_expand/readme/CONTRIBUTORS.rst (hunk 3 above)
Comment (BT-aestebanez) [reply web_group_expand/readme/CONTRIBUTORS.rst]: done, thanks

File: web_group_expand/__manifest__.py [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
     "category": "Web",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
     "license": "AGPL-3",
     "author": "OpenERP SA, "
     "AvanzOSC, "
     "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
     "Therp BV, "
     "Xtendoo, "
+    "Brain-tec, "
```
Comment (simahawk) [inline web_group_expand/__manifest__.py]: you can list yourself as contributors, not author as the migration has little impact here.

Comment (BT-aestebanez) [reply web_group_expand/__manifest__.py]: Thanks for the review.
I agree.
Done!

**Review by etobella — APPROVED** [review]
LGTM :+1: 
Functional review

**Comment by BT-fgarbely** [comment]
Hello @simahawk 

What's the state here? Can this be merged?
Thanks.

**Comment by simahawk** [comment]
@BT-fgarbely to me is fine but pre-commit isn't happy. :wink:

**Comment by BT-fgarbely** [comment]
Hello @BT-aestebanez 

Can you have a look to the pre-commit problems?

**Comment by BT-aestebanez** [comment]
Thanks for the comments :)

**Pull request merged**